exchanges/validate/README.md

```diff
@@ -0,0 +1,112 @@
+# GoCryptoTrader package Validate
```

Comment:
Shouldn't this be generated via the documentation tool?

Reply:
Yes, I was lazy.